### ISSUE TITLE
[Feature Request] Add option in budget settings to set specific spending days (e.g only Mon - Fri)

### ISSUE THREAD

**Issue by l2eused** [jameskokoska/Cashew#409]
## Is your feature request related to a problem?

Yes, I'm student and I using Cashew to track my expense in school week days (**5 days in a week**).

Now I already problem about what exactly amounts I can spend in a day. Because in **Cashew Budget**, it's **showing amounts based on 7 days** not 5 days as I use (in weekend I didn't use my own budget but family). So the text sentence "*You can spend .../day for ... more days*" will **showing money that I can spend lower than it should be** for me.

## Describe the solution you'd like

The best solution for my case is **adding option in Edit Budget**.

In example:
```
Edit Budget:
- Name: Budget
- Amounts: 300$/1 month
- Beginning: August 1
- **Day(s) per week: 5 days**
```

In above example, adding days per week option to set calculate available spend per day from 7 (normal per week) to user setting.

That doesn't effect any expense in weekend, They still adding to budget normally. Just use that setting for calculate spend per day display sentence only. Because user can exclude unwanted expense from budget in each expense adding already.

> TLDR; Allow user to set **You can spend .../day for ... more days** from normally 7 days per week to user preference (i.e 5 days per week)

**Comment by jameskokoska** [comment]
Hi there! I understand your use case but implementing this feature could create confusion within the UI. There is complexity in how an interface supports this customization and justifying the per day spending being different from the available days on the entire budget. Additionally, the budget cycles don't support gaps, so the cycle would need to be continuous. Because of this, I don't see how this feature would make sense within this app unfortunately.
